### ISSUE TITLE
Refactor GPS minimum tests in Attitude module

### ISSUE THREAD

**Issue by kubark42** [TauLabs/TauLabs#2192]
There are bunches of 
`if ((gpsData.Satellites >= 6) && (gpsData.PDOP <= 4.0f))`

and
`if ((gpsData.Satellites < 6) && (gpsData.PDOP > 4.0f))`

and
`if ((gpsData.Satellites >= 7) && (gpsData.PDOP <= 3.5f))`

in at least:
- `Modules/Attitude/attitude.c`
- `Modules/GPS/GPS.c`

These should be grouped into a simple test function (or two functions).

**Comment by peabody124** [comment]
Agreed about consolidating. Just let's keep some kind of hysteresis.
